linux.py

```diff
@@ -28,11 +28,8 @@ def get_active_window_raw():
     if match != None:
         ret = match.group("name").strip(b'"')
         #print(type(ret))
-        '''
-        ret is str for python2
-        ret is bytes for python3 (- gives error while calling in other file)
-        be careful
-        '''
+        if sys.version[0] == '3': # if compiler is python 3
```

Comment:
Hello @NumberPiOso,

[The documentation](https://docs.python.org/3/library/sys.html#sys.version) says that you shouldn't use `sys.version` like this.
> Do not extract version information out of it, rather, use [version_info](https://docs.python.org/3/library/sys.html#sys.version_info) and the functions provided by the [platform](https://docs.python.org/3/library/platform.html#module-platform) module.

```suggestion
        if sys.version_info.major > 2:  # Python version is 3 or later
```

Reply:
Completely agreed.